stable/mariadb/templates/NOTES.txt

```diff
@@ -1,66 +1,35 @@
-{{- if contains .Values.service.type "LoadBalancer" }}
-{{- if not .Values.usePassword }}
--------------------------------------------------------------------------------
- WARNING
 
-    By specifying "service.type=LoadBalancer" and "usePassword=false" you have
-    most  likely exposed the MariaDB service externally without any authentication
-    mechanism.
+Please be patient while the chart is being deployed
 
-    For security reasons, we strongly suggest that you switch to "ClusterIP" or
-    "NodePort". As alternative, you can also switch to "usePassword=true"
-    providing a valid pasword on "mariadbRootPassword" parameter.
+Tip:
 
--------------------------------------------------------------------------------
-{{- end }}
-{{- end }}
-
-** Please be patient while the chart is being deployed **
-
-MariaDB can be accessed via port 3306 on the following DNS name from within your cluster:
-{{ template "mariadb.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
-
-{{ if .Values.usePassword -}}
-To get the root password run:
+  Watch the deployment status using the command: kubectl get pods -w --namespace {{ .Release.Namespace }}
```

Comment:
While you're at, also add `-l app=mariadb` to that commandline :)

Reply:
I will put the release label, it will be more exact